internal/fields/validate.go

```diff
@@ -320,8 +322,15 @@ func (v *Validator) validateDocumentValues(body common.MapStr) multierror.Error
 				continue
 			}
 
+			var actualExpectedDatasets []string
+			for _, dataset := range v.expectedDatasets {
+				renderedDataset := mustache.Render(dataset, body)
```

Comment:
Could you add a test case that would leverage this rendering? A good place may be here https://github.com/elastic/elastic-package/blob/main/test/packages/parallel/awsfirehose/data_stream/log/routing_rules.yml

cc @kaiyan-sheng

Reply:
Yep, definitely!

Reply:
@jsoriano I added an integration test to check the expression rendering, but the test is [failing](https://buildkite.com/elastic/elastic-package/builds/1289#0189fedd-3a84-43a8-8258-d6fa49c837cc). What's the best way to run this locally?

Reply:
The closest match I could find is the following:

```shell
PACKAGE_UNDER_TEST=kubernetes PACKAGE_TEST_TYPE=parallel ./scripts/test-check-packages.sh 
```

However, the previous command runs fine on my machine. 

There's something else, I'm looking into this.

Reply:
Nevermind! 

I didn't run `elastic-package check` in the kubernetes test package. My bad! 🤦